.npmignore

```diff
@@ -24,7 +24,10 @@ build/Release
 
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
-node_modules
+node_modules/
 examples/biz-website/public/
 examples/blog/public/
 *.un~
+
+# IDEA/WebStorm configuration directory
+.idea/
```

Comment:
I'm not sure, I usually don't include it as not to pollute the project with unnecessary IDE configuration files